evidence.go

```diff
@@ -0,0 +1,279 @@
+package ccatoken
+
+import (
+	"crypto"
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"strings"
+
+	cose "github.com/veraison/go-cose"
+	ptoken "github.com/veraison/psatoken"
```

Comment:
is this aliasing really worth? one normally shouldn't drop bits from an acronym, the information loss can be fatal :-)

Reply:
ok, strictly not required, so changed it!